docs/3.api/1.composables/use-fetch.md

```diff
@@ -72,6 +73,7 @@ All fetch options can be given a `computed` or `ref` value. These will be watche
   * `transform`: a function that can be used to alter `handler` function result after resolving
   * `pick`: only pick specified keys in this array from the `handler` function result
   * `watch`: watch an array of reactive sources and auto-refresh the fetch result when they change. Fetch options and URL are watched by default. You can completely ignore reactive sources by using `watch: false`. Together with `immediate: false`, this allows for a fully-manual `useFetch`.
+  * `shallow`: return data in a shallow ref, instead of a deep ref object. This is useful when you want to fine-tune reactivity.
```

Comment:
```suggestion
  * `shallow`: return data in a shallow ref, instead of a deep ref object. This can improve performance when your data does not need to be deeply reactive.
```